Symfony/CS/AbstractFunctionReferenceFixer.php

```diff
@@ -0,0 +1,123 @@
+            // this expression is differs from expected, resume
+            return self::find($functionNameToSearch, $tokens, $openParenthesis, $end);
+        }
+
+        // second criteria check: ensure namespace is the root one
+        if ($functionNamePrecedingToken->isGivenKind(T_NS_SEPARATOR)) {
+            $namespaceCandidate = $tokens->getPrevMeaningfulToken($functionNamePrefix);
+            $namespaceCandidateToken = $tokens[$namespaceCandidate];
+            if ($namespaceCandidateToken->isGivenKind(array(T_NEW, T_STRING, CT_NAMESPACE_OPERATOR))) {
+                // here can be added complete namespace scan
+                // this expression is differs from expected, resume
+                return self::find($functionNameToSearch, $tokens, $openParenthesis, $end);
+            }
+        }
+
+        // final step: find closing parenthesis
+        $closeParenthesis = $tokens->findBlockEnd(Tokens::BLOCK_TYPE_PARENTHESIS_BRACE, $openParenthesis);
+        if (null === $closeParenthesis) {
+            // this sequence is not closed, try resuming
+            return self::find($functionNameToSearch, $tokens, $openParenthesis, $end);
+        }
+
+        return array($functionName, $openParenthesis, $closeParenthesis);
+    }
+
+    /**
+     * Count amount of parameters in a function/method reference.
+     *
+     * @param int                                  $openParenthesis
+     * @param int                                  $closeParenthesis
+     * @param Tokens|\Symfony\CS\Tokenizer\Token[] $tokens
+     *
+     * @return int
+     */
+    protected function countArguments($openParenthesis, $closeParenthesis, Tokens $tokens)
+    {
+        $firstSensibleToken = $tokens->getNextMeaningfulToken($openParenthesis);
+        if (')' === $tokens[$firstSensibleToken]->getContent()) {
+            return 0;
+        }
+
+        $argumentsCount = 1;
+        for ($paramContentIndex = $openParenthesis + 1; $paramContentIndex < $closeParenthesis; ++$paramContentIndex) {
+            // skip nested (...) constructs
+            if ('(' === $tokens[$paramContentIndex]->getContent()) {
+                $paramContentIndex = $tokens->findBlockEnd(Tokens::BLOCK_TYPE_PARENTHESIS_BRACE, $paramContentIndex);
+                continue;
+            }
+
+            // if comma matched, increase arguments counter
+            if (',' === $tokens[$paramContentIndex]->getContent()) {
+                ++$argumentsCount;
+            }
+        }
+
+        return $argumentsCount;
+    }
+
+    /**
+     * Checks if function/method defined in the scope.
+     *
+     * @param string $functionName
+     * @param Tokens $tokens
+     *
+     * @return bool
+     */
+    protected function isDefinedInScope($functionName, Tokens $tokens)
+    {
+        $definitionSequence = array(array(T_FUNCTION, 'function'), array(T_STRING, $functionName));
```

Comment:
And if that function is defined in same scope but in different file - this method will return false negative.
Because the fixer don't know whole imported files (which may be even some vendor files outside the repo).